app-starters-common/pom.xml

```diff
@@ -24,4 +24,12 @@
 		<module>app-starters-twitter-common</module>
 	</modules>
 
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-java-dsl</artifactId>
+			<version>${spring-integration-java-dsl.version}</version>
+		</dependency>
```

Comment:
To be more clear I don't understand why we are here in apps worry about the SCSt compatibility with its Boot `1.3.x`. My point is:
- we develop microservices, so it doesn't matter on which codebase we are based. It is just service with its own independent container in the separate JVM. That's a bit weird that we are tied with some other release. Spring Boot 1.4 isn't around the corner with the SI-4.3 foundation. So, I'd like to raise a topic as soon as possible. (Agree this PR might not be the best place)
- What is the reason to stay with the Maven if we still should spread all these versions (and deps) everywhere and don't have any separations gain. The Gradle would do the same, but with less wordy way :smile: 

Just my five cents